### PR TITLE
HDS-258 added file_format param to FhirSender and AutoMapperToFhir

### PR DESCRIPTION
**Pull request by changsu-bWell**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -50,11 +50,11 @@ services:
   fhir:
     depends_on:
       - mongo
-    image: imranq2/node-fhir-server-mongo:3.0.35
+    image: imranq2/node-fhir-server-mongo:3.1.48
```
Comment (changsu-bWell) [inline docker-compose.yml:53]: `3.1.48` is the one we use in helix.pipelines repo

File: spark_pipeline_framework/transformers/fhir_sender/v1/fhir_sender.py [hunk 2]
```diff
@@ -423,12 +430,16 @@ def _transform(self, df: DataFrame) -> DataFrame:
                         self.logger.info(
                             f"Executing requests and writing FHIR {resource_name} responses to disk..."
                         )
-                        result_df.write.mode(mode).json(str(response_path))
+                        result_df.write.mode(mode).format(file_format).save(
+                            str(response_path)
+                        )
```
Comment (changsu-bWell) [inline spark_pipeline_framework/transformers/fhir_sender/v1/fhir_sender.py:435]: just updated as the "generalized" form

File: spark_pipeline_framework/transformers/send_automapper_to_fhir/v1/automapper_to_fhir_transformer.py [hunk 3]
```diff
@@ -190,15 +198,27 @@ def _transform(self, df: DataFrame) -> DataFrame:
...
-                    ).transform(df)
+                    if file_format == "parquet":
+                        FrameworkParquetExporter(
+                            name=f"{name}_parquet_exporter",
+                            parameters=parameters,
+                            progress_logger=progress_logger,
+                            view=view,
+                            mode=mode,
+                            file_path=fhir_resource_path,
+                        ).transform(df)
```
Comment (changsu-bWell) [inline spark_pipeline_framework/transformers/send_automapper_to_fhir/v1/automapper_to_fhir_transformer.py:211]: I found that there is already , `FrameworkParquetExporter` transformer, so just used it here when it's parquet

File: docker-compose.yml (hunk 1 above)
Comment (changsu-bWell) [reply docker-compose.yml]: reverted to prev v - 3.0.35

**Review by cdukebwell — APPROVED** [review]

**Pull request merged**